[
  {
    "StatusHeading": "Known power outage - working to restore power",
    "Area": "Lugarno",
    "Cause": "Under investigation",
    "Customers": "56",
    "EstRestTime": "2024-03-22T19:30:00",
    "StartDateTime": "2024-03-22T15:35:00",
    "WebId": 101003,
    "Status": 1
  }
]
